composeApp/src/commonMain/kotlin/designsystem/login/OrDivider.kt

```diff
@@ -0,0 +1,43 @@
+package designsystem.login
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import designsystem.colors.TextColor
+
+@Composable
+fun LoginDivider() {
```

Comment:
Should it be in the footer as well? Just a question.

We also need to add a modifier here as argument.

```kotlin
@Composable
fun LoginDivider(modifier: Modifier = Modifier) {
...
```

Just to make flexible

Reply:
- Not sure it applies 1:1 there since this one has the "OR" text between the two dividers on the same row
- I'll add the modifier

Reply:
Done